device-types/HPE/Aruba-EC-L-H.yaml

```diff
@@ -1,47 +1,52 @@
 ---
 manufacturer: HPE
-model: Aruba ec-l-h
+model: Aruba EC-L-H
 slug: hpe-aruba-ec-l-h
 part_number: '201754'
 u_height: 1
 is_full_depth: true
 airflow: front-to-rear
+weight: 32.0
+weight_unit: lb
+front_image: true
+rear_image: true
 console-ports:
   - name: console
     type: de-9
     label: console
 power-ports:
```

Comment:
Hi Thank you for your contribution.
Are these modular PSUs? If so they should be modeled as `module-bays` with the required PSUs added to the modules folder if they don't already exist.